flow.js

```diff
@@ -103,7 +103,7 @@ class Flow {
         {condition: 'windows', nextStep: 'repair-window-problems'},
         {condition: 'dampOrMould', nextStep: 'bathroom-damp-mould-problems'},
         {condition: 'damagedOrStuckDoors', nextStep: 'repair-door-problems'},
-        {condition: 'showerIncludingTrayAndDoor', nextStep: 'repair-shower-problems'},
+        {condition: 'showerIncludingTrayAndDoor', nextStep: 'unable-to-book'},
```

Comment:
Guessing your tests fail because it goes to a different step than expected, hence those 4sec timeout errors upon trying to find the element.